### PR TITLE
Feature package manifests

### PR DESCRIPTION
**Pull request by iamjoshellis**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "lighthouse-scss-lint",
```
Comment (cjquinn) [inline bower.json]: Needs a better name :)

File: package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "lighthouse-scss-lint",
```
Comment (cjquinn) [inline package.json]: A BETTER NAME JOSH 🎱

**Review by cjquinn — CHANGES_REQUESTED** [review]

**Comment by cjquinn** [comment]
So are we changing the repo name to `scss-linthouse` or what?

**Comment by iamjoshellis** [comment]
Ah I was thinking it would be better for it to be namespaced. e.g.
`linthouse-scss`
`linthouse-js`
`linthouse-html`
`linthouse-php`
Keep all of the lighthouse linting stuff grouped in a manifest file or when search repos. Whaddya think?

**Review by cjquinn — APPROVED** [review]

**Pull request merged**